packages/block-editor/src/components/block-list/block.js

```diff
@@ -451,8 +451,17 @@ const applyWithDispatch = withDispatch( ( dispatch, ownProps, registry ) => {
 					}
 
 					moveFirstItemUp( rootClientId );
-				} else {
-					removeBlock( clientId );
```

Comment:
Should we maybe just not do anything? I'm not sure if pressing Backspace at the start of a heading should convert it to a paragraph? Are there any editors that do this?

Reply:
Google Docs and MS Word don't seem to transform a heading into a paragraph. Regarding Notion, it seems to transform a heading into a paragraph.

https://github.com/WordPress/gutenberg/assets/54422211/1c38ba09-6f18-4abe-913a-85e754ed2021

Personally, I like the current behavior of the Backspace key. If the backspace key does nothing, we will need to transform the existing heading into a paragraph or delete it in order to start a new article again from a paragraph. This seems a little cumbersome to me.